runtime/parachains/src/dmp.rs

```diff
@@ -204,7 +205,10 @@ impl<T: Config> Pallet<T> {
 				.saturating_div(config.max_downward_message_size)
 				.saturating_div(2u32)) as usize
 		{
-			Self::increment_fee_factor(para);
+			let message_size_factor =
+				FixedU128::from_inner(serialized_len.saturating_div(1024) as u128)
+					.saturating_mul(MESSAGE_SIZE_FEE_BASE);
```

Comment:
You'd be doing `1.001 * msg_in_kb` here, which would make the number really huge if it's 5 KB. I think you really mean to do `1 + 0.001 * msg_in_kb` instead?